packages/python-packages/api-stub-generator/apistub/_stub_generator.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import glob
+import sys
+import os
+import argparse
+import astroid
+import inspect
+import io
+import importlib
+import json
+import logging
+import shutil
+import typing
+from subprocess import check_call
+import zipfile
+
+
+from ._apiview import ApiView, APIViewEncoder, Navigation, Kind, NavigationTag
+
+INIT_PY_FILE = "__init__.py"
+
+logging.getLogger().setLevel(logging.INFO)
+
+class NodeIndex:
+    """Maintains name to navigation ID"""
+    index = {}
+
+    def add(self, name, node):
+        if name in self.index:
+            raise ValueError("Index already has {} node".format(name))
+        self.index[name] = node
+
+    def get(self, name):
+        return self.index.get(name, None)
+
+    def get_id(self, name):
+        node = self.get(name)
+        if node and hasattr(node, "namespace_id"):
+            return node.namespace_id
+        return None
+
+class StubGenerator:
+
+    def _find_modules(self, pkg_root_path):
+        """Find modules within the package to import and parse
+        :param str: pkg_root_path
+            Package root path
+        :rtype: list
+        """
+        modules = []
+        azure_root_path = os.path.join(pkg_root_path, "azure")
+        for root, subdirs, files in os.walk(azure_root_path):
+            # Ignore any modules with name starts with "_"
+            # For e.g. _generated, _shared etc
+            dirs_to_skip = [x for x in subdirs if x.startswith("_")]
+            for d in dirs_to_skip:
+                subdirs.remove(d)
+
+            # Add current path as module name if _init.py is present
+            if INIT_PY_FILE in files:
+                module_path = root.replace(pkg_root_path, "")
+                module_name= module_path.replace(os.path.sep, ".")[1:]
+                modules.append(module_name)
+
+                # Add any public py file names as modules
+                sub_modules = [os.path.splitext(os.path.basename(f))[0] for f in files if f.endswith(".py") and not os.path.basename(f).startswith('_')]
+                modules.extend(["{0}.{1}".format(module_name, x) for x in sub_modules])
+
+        logging.info("Modules in package: {}".format(modules))
+        return modules
+
+
+    def _generate_tokens(self, pkg_root_path, package_name, version):
+        """This method returns a dictionary of namespace and all public classes in each namespace
+        """
+        # Import ModuleNode since it needs NodeIndex.
+        # Importing it globaly can cause circular dependency
+        from apistub.nodes._module_node import ModuleNode
```

Comment:
This looks a tiny bit weird to me, can you adjust this to `nodes._module_node`?

Reply:
Reason I ask is because at the top of the file we import with the relative method. 
```
from ._apiview import ApiView...
```

So if we did THAT the same we we do this one, we'd write it `from apistub._apiview import ApiView...` which is a little bit odd. It's not necessarily WRONG to my knowledge, just makes me tilt my head to one side.